Sources/Afterpay/Info/InfoWebViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  InfoWebViewController.swift
+//  Afterpay
+//
+//  Created by Ryan Davis on 7/8/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+final class InfoWebViewController: UIViewController, WKNavigationDelegate {
+
+  private let infoURL: URL
+
+  private var webView: WKWebView { view as! WKWebView }
+
+  init(infoURL: URL) {
+    self.infoURL = infoURL
+
+    super.init(nibName: nil, bundle: nil)
+  }
+
+  override func loadView() {
+    let webView = WKWebView()
+    webView.allowsLinkPreview = false
+    webView.navigationDelegate = self
+    view = webView
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    if #available(iOS 13.0, *) {
+      overrideUserInterfaceStyle = .light
+    } else {
+      navigationItem.rightBarButtonItem = UIBarButtonItem(
+        title: "Cancel",
```

Comment:
We might prefer `"Close"` in this context

```suggestion
        title: "Close",
```

Reply:
Good idea 👍